girder/models/user.py

```diff
@@ -85,6 +90,10 @@ def validate(self, doc):
             # an email address from a login
             raise ValidationException('Login may not contain "@".', 'login')
 
+        if 'hashAlg' in doc:
```

Comment:
Should we add some form of structured comment in the event that we implement #946?

Reply:
The simple existence of your comment here just created a reference on that issue page :astonished: